.github/workflows/stackaid.yml

```
@@ -0,0 +1,11 @@
+name: 'fund-on-stackaid'
+on:
+  push:
+    branches:
+      - main
+jobs:
+  stackaid-json:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: stackaid/generate-stackaid-json@v1.9
```

We pin actions in this repo. And checkout@v2 uses outdated Node, triggering notices.
```suggestion
      - uses: actions/checkout@v3
      - uses: stackaid/generate-stackaid-json@bbccf9adf6ab4acd69aa979b0cdc7e93a5e83d78 # pin@v1.9
```

---

Thanks for the feedback @vaind We'll update our README as well.